libvirt.go

```diff
@@ -449,22 +520,17 @@ func (l *Libvirt) Events(dom string) (<-chan DomainEvent, error) {
 	return c, nil
 }
 
-// Migrate synchronously migrates the domain specified by dom, e.g.,
+// DomainMigrate synchronously migrates the domain specified by dom, e.g.,
 // 'prod-lb-01', to the destination hypervisor specified by dest, e.g.,
 // 'qemu+tcp://example.com/system'. The flags argument determines the
 // type of migration and how it will be performed. For more information
 // on available migration flags and their meaning, see MigrateFlag*.
-func (l *Libvirt) Migrate(dom string, dest string, flags MigrateFlags) error {
+func (l *Libvirt) DomainMigrate(d *Domain, dest string, flags MigrateFlags) error {
```

Comment:
Sorry this is not suitable for me, because in this case i have unnecessary lookups all the time and in my case i can have two domains with the same name (that does not forbidden in libvirt because of uuid). So if this is not possible with this project - i'm maintain my own fork, sorry.